cmd/tailscale/cli/up.go

```diff
@@ -117,7 +120,14 @@ func newUpFlagSet(goos string, upArgs *upArgsT) *flag.FlagSet {
 		upf.BoolVar(&upArgs.forceDaemon, "unattended", false, "run in \"Unattended Mode\" where Tailscale keeps running even after the current GUI user logs out (Windows-only)")
 	}
 	upf.DurationVar(&upArgs.timeout, "timeout", 0, "maximum amount of time to wait for tailscaled to enter a Running state; default (0s) blocks forever")
-	registerAcceptRiskFlag(upf, &upArgs.acceptedRisks)
+
+	if cmd == "up" {
```

Comment:
not doing the `--up=false` thing for "login", eh? (even with all the prefs flags too)

Reply:
I was going to get back to this as it requires some a bunch more testing on when the login happens and when we run stop.